specification/devcenter/DevCenter/examples/2023-04-01/DevBoxes_ListAllDevBoxesByUser.json

```diff
@@ -1,9 +1,9 @@
 {
   "title": "Lists Dev Boxes in the Dev Center for a particular user.",
-  "operationId": "DevCenter_ListAllDevBoxesByUser",
+  "operationId": "DevBoxes_ListAllDevBoxesByUser",
```

Comment:
Same question as above

Reply:
see comment above, removed for accurate/consistency